kbase-extension/static/kbase/js/common/data.js

```diff
@@ -5,6 +5,9 @@ define(['kb_service/client/workspace', 'kb_service/utils', 'common/runtime'], (
 ) => {
     'use strict';
     function filterObjectInfoByType(objects, types) {
+        if (types.includes('*')) {
```

Comment:
This is the function that returns data from the data list to display in the dropdown input. If the types list has "*" as a string in it, then it'll just return everything.